src/storage/datasets_index.rs

```diff
@@ -53,6 +108,8 @@ impl DatasetsIndex {
 pub fn parse_assignment(assignment: WorkerAssignment) -> anyhow::Result<(ChunkSet, DatasetsIndex)> {
```

Comment:
Is this function still used? I think it was used for parsing pongs

Reply:
removed